packages/atlas/src/providers/snackbars/snackbar.tsx

```diff
@@ -42,7 +42,7 @@ export const Snackbars: React.FC = () => {
     <SnackbarsContainer>
       <TransitionGroup>
         {snackbars.map(({ id, iconType, onActionClick, onExit, ...snackbarProps }) => (
-          <CSSTransition key={id} timeout={parseInt(cVar('animationTimingFast', true)) * 2} classNames="snackbar">
+          <CSSTransition key={id} timeout={parseInt(cVar('animationTimingMedium', true)) * 2} classNames="snackbar">
```

Comment:
Shouldn't we set here `animationTransitionCallout`?

Reply:
But it's property, `animationTransitionCallout` uses `animationTimingMedium`

Reply:
oh, you're right!